### PR TITLE
create the OrderItem model

### PR DESCRIPTION
**Pull request by MahmoudMDarabaih**

### CODE REVIEW

File: src/models/OrderItem.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    price: {
+      type: DataTypes.DOUBLE,
+      allowNull: false,
+    },
+  },
+  {
+    sequelize,
+    modelName: "OrderItem",
+    tableName: "OrderItem",
```
Comment (AhmadAlBarasy) [inline src/models/OrderItem.ts]: Please pluralize the name of the table bro ;)
and also make sure the name starts with a lowercase character

File: src/models/OrderItem.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Model, DataTypes } from "sequelize";
+import sequelize from "../database";
+
+class OrderItem extends Model {}
+
+OrderItem.init(
+  {
```
Comment (AhmadAlBarasy) [inline src/models/OrderItem.ts:7]: where is the id for this table?

**Review by AhmadAlBarasy — CHANGES_REQUESTED** [review]
Please check the comments I provided on the code

File: src/models/OrderItem.ts (hunk 1 above)
Comment (rshgithub) [reply src/models/OrderItem.ts]: > Please pluralize the name of the table bro ;) and also make sure the name starts with a lowercase character

the table better to be named : tableName: 'order_items',
also he you can add :  underscored: true,   which converts the " camelCase " to " snake_case "

File: src/models/OrderItem.ts (hunk 2 above)
Comment (rshgithub) [reply src/models/OrderItem.ts:7]: add this :
id: {
  type: DataTypes.UUID,
  defaultValue: DataTypes.UUIDV4,
  primaryKey: true,
  allowNull: false,
  },

**Review by rshgithub — CHANGES_REQUESTED** [review]

File: src/models/OrderItem.ts (hunk 1 above)
Comment (MahmoudMDarabaih) [reply src/models/OrderItem.ts]: done

File: src/models/OrderItem.ts (hunk 2 above)
Comment (MahmoudMDarabaih) [reply src/models/OrderItem.ts:7]: done

**Review by AhmadAlBarasy — APPROVED** [review]
محرقة يابا زعامة معلم احرق يايا

**Review by rshgithub — APPROVED** [review]

**Pull request merged**